deploy/cephcsi/image/Dockerfile

```diff
@@ -0,0 +1,19 @@
+
+FROM centos:7
+LABEL maintainers="Kubernetes Authors"
+LABEL description="Ceph-CSI Plugin"
+
+ENV CEPH_VERSION "mimic"
+
+RUN yum install -y centos-release-ceph && \
```

Comment:
Wait a second, I just looked into that more closely and it seems that `centos-release-ceph` installs... Luminous!
And then `CEPH_VERSION` is set to `mimic`, so that's wrong and confusing.

We are not even pulling packages from download.ceph.com. So I'm taking back what I said and urge us to use `ceph/ceph` instead. I have a PR already to add the missing packages here: https://github.com/ceph/ceph-container/pull/1337

Reply:
I changed to be based on ceph/ceph:v14. I wasn't entirely sure if that was the best version but I'd figured I'd just hear from you otherwise.

Reply:
Thanks 14.2.1 is around the corner just a few more days.